src/app/designpad/elements/tempTransition.ts

```diff
@@ -0,0 +1,17 @@
+import {Transition} from "./transition";
+import * as P5 from 'p5';
+import {BasicState} from "./basicState";
+import {Triple} from "./triple";
+
+
+export class TempTransition extends Transition {
```

Comment:
What are you using this for?

Reply:
It draws the transition while placing it on the sketch, so instead of drawing it from state to state, it draws it from state to mouse point.